modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/data_set.jsp

```diff
@@ -22,15 +22,15 @@ renderResponse.setTitle(ParamUtil.getString(request, "dataSetLabel"));
 		HashMapBuilder.<String, Object>put(
 			"backURL", backURL
 		).put(
-			"dataSetERC", ParamUtil.getString(request, "dataSetERC")
+			"clientExtensionCellRenderers", fdsViewsDisplayContext.getFDSCellRendererCETsJSONArray()
```

Comment:
I'd say `fdsClientExtensionCellRenderers`  and `fdsFilterClientExtensions` names can be considered still ok, because they represent CX that only FDS can understand/inject. They belong to FDS jargon though we use DSM to configure them.

I don't think this is very important, however, CX type won't likely be changed (`FDSCellRendererCET.java` and friends) so we're introducing a small inconsistency if we remove `fds` prefix

Reply:
Yeah, that was my line of reasoning. Backend tied to way CX is named, abstraction in frontend. If we ever change `FDS` to new `Mango` component, we can change the line to 
```js
"clientExtensionCellRenderers", fdsViewsDisplayContext.getMangoCellRendererCETsJSONArray()
```
and keep frontend code as it is.